packages/http-cors/index.d.ts

```diff
@@ -2,6 +2,7 @@ import middy from '../core'
 
 interface ICorsOptions {
   origin: string;
+  origins: string[];
```

Comment:
this should be optional: `origins?: string[]` - to match up with the javascript: https://github.com/middyjs/middy/pull/195/files#diff-3f35fefc85b62544f23244fe1407a252R10

The current ICorsOptions interface is incorrect - I have a fix for that about to be submitted to PR - so no need to fix anything other than this one line imho.

Reply:
Then let's wait for the other PR to be merged and then we'll get this one updated (if needed) and merged too. What do you think? :)

Reply:
If 200 lands, then this won't pass the ci check, if this lands first then 200 will fail because the tests do not look for this value being optional. So I don't think we need to be too strategic around which is merged first :) 

I called it out primarily as I noticed it and wanted to let @thejuan know about my impending PR and how it may affect this work.

Reply:
yeah, this branch should be rebased off master and then any conflicts and test errors should be fixed.